runtimes/huggingface/mlserver_huggingface/common.py

```diff
@@ -38,12 +38,17 @@ class Config:
         env_prefix = ENV_PREFIX_HUGGINGFACE_SETTINGS
 
     task: str = ""
+    task_suffix: str = ""
```

Comment:
Wouldn't it be enough to include the suffix in the `task` field?

Reply:
Sure, but for other reasons.

Some default metadatas is associated with task types. 

https://github.com/SeldonIO/MLServer/blob/c07a1702581abd348fb7c30fd7e8c663b3505bb2/runtimes/huggingface/mlserver_huggingface/metadata.py#L179

The task field does not specify whether it represents the task type or the task name. 

Maybe there is a better way. This only solves the problem I have encountered

Reply:
Right, I see... Thanks for clarifying that.

Do you know whether hugginface uses other variations of the task name besides the suffix? If that's the case, we could go for something more generic, like regexes for the metadata dict keys? 

If, on the other hand, it's only suffixes - then I think it should be alright to just add this extra key to the config.

Reply:
PS: Could we also add a note as to why we need `task_suffix`, perhaps referencing the original issue?

Reply:
From the comment in transforms lib. https://github.com/huggingface/transformers/blob/6767ce71d661ae57afa38b6cd0d533e3dfd463fa/src/transformers/pipelines/__init__.py#L257

This is a special case for task `translation`.